docs/en/supported_models/supported_models.md

```diff
@@ -72,6 +72,7 @@ The TurboMind engine doesn't support window attention. Therefore, for models tha
 |  DeepSeek-MoE  |     16B     | LLM  |    Yes    |   No    |   No    |  No  |  No   |
 |  DeepSeek-V2   |  16B, 236B  | LLM  |    Yes    |   No    |   No    |  No  |  No   |
 |    MiniCPM3    |     4B      | LLM  |    Yes    |   Yes   |   Yes   |  No  |  No   |
+| MiniCPM-V-2_6  |     8B      | LLM  |    Yes    |   No    |   No    |  No  |  Yes  |
```

Comment:
could update supported_models in readme as well
https://github.com/InternLM/lmdeploy?tab=readme-ov-file#supported-models

Reply:
MiniCPM-V-2_6 already exists in README， because turbomind engine supported it before, and  support list in the README does not distinguish between the turbomind and pytorch engine.